apps/gro/controllers/confirm.js

```diff
@@ -27,32 +28,16 @@ ConfirmController.prototype.saveValues = function saveValues(req, res, callback)
 
     var subject = locali18n.translate('pages.email.information.subject');
 
-    var d = {
-      values: (function getSessionValues(data) {
-        var r = {};
-        for (var prop in data) {
-          r[prop] = data[prop];
-        }
-        return r;
-      }(req.sessionModel.attributes))
-    };
+    var d = {values: _.pick(req.sessionModel.attributes, _.identity)};
```

Comment:
yeah we dont want to be altering the sessionModel, Object assign will give the same result as the current code, was just thinking if we need to deepClone for future possible nested data, but we are just emailing the form values so probably not. Ideally we would selectively add the fields/values as sessionModel contains a load of other suff that isn't needed but this way (copying everything) is simpler